### DISCUSSION TITLE
how to install new version on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by chris-colinsky** [liquidctl/liquidctl#640] [Q&A]
Ubuntu 22.04 installs an older version (1.5.2) which does not support my particular device: Corsair Hydro H115i Pro (requires 1.9.1).  Is there a way to install the version needed for my AIO on 22.04?  Thank you.

**Comment by Serphentas** [comment]
I see an even older version (1.5.1) here: https://packages.ubuntu.com/jammy/liquidctl. Anyway, you can try the manual installation which is basically:
- getting dependencies
- installing via pip
- ensuring access right are correct

**Reply by chris-colinsky** [reply]
I got 1.12.1 working via pip install, but I'm still having issues getting that setup to run as a systemd service.  get's a permission error on system startup, but runs fine if I manually start up the service once the system is fully booted.

The error in the systemd logs:
systemd[1]: Starting AIO startup service...
liquidctl[1940]: ERROR: Corsair Hydro H115i Pro: insufficient permissions
systemd[1]: liquidcfg.service: Main process exited, code=exited, status=1/FAILURE
systemd[1]: liquidcfg.service: Failed with result 'exit-code'.
systemd[1]: Failed to start AIO startup service.

**Reply by Serphentas** [reply]
Can you confirm whether the udev rules are installed correctly and that your user is in the plugdev group ?

**Reply by chris-colinsky** [reply]
To wrap this up, I did get it working as a systemd service. although I could use liquidctl when logged in as me and everything worked as expected, my assumption was that it was setup correctly.  I did try to set the User in the service file but that didn't work, still got permission error in logs.  I then ran the pip install using sudo and after the next boot the service worked as expected and my setting we're applied automatically after boot.
